airbyte-container-orchestrator/src/main/resources/application.yml

```diff
@@ -0,0 +1,6 @@
+micronaut:
+  server:
+    port: 9000
+
+airbyte:
+  config-dir: /config
```

Comment:
Is this really the only config value/env var that the container orchestrator needs to run?  I would expect to see a bunch of the configuration from airbyte-workers in here, as they share some similar code.  If the plan is to make a second pass to remove references to `EnvConfigs`, then disregard this comment.

Reply:
You are correct, the plan is to take a second pass to remove `EnvConfigs`.  I initially went down the path of doing that in this PR but decided against it as it increased the impact of this change to a size that I wasn't comfortable with.